tests/compile.sh

```diff
@@ -132,29 +132,26 @@ fi
 set -ex
 
 # Valid applications
-if [[ "${MAKE_OPT}" == *"APP=ATM"* ]]; then
+if [[ "${MAKE_OPT}" == *"APP=ATMAERO"* ]]; then
+    CMAKE_FLAGS="${CMAKE_FLAGS} -DAPP=ATMAERO"
+    unset PYTHONPATH
```

Comment:
What is the need to unset the PYTHONPATH here?

Reply:
Python 3.x is required to build GOCART. On Hera, using `intelpython/3.6.8` leads to the following error:
```
ImportError: Unable to import required dependencies:
numpy:
```
since the Python packages included in `PYTHONPATH` are not compatible with intelpython, which already includes the required dependencies.

Reply:
Argh, would it be better - if possible - to fix `intelpython/3.6.8` ?

Reply:
What is python being used for in GOCART?

Reply:
Python is required for automatic code generation, to create include files for the GOCART components and its subcomponents.

Why do we need to "fix" intelpython/3.6.8? It is a standard, self-contained Python installation.

Reply:
Hera support does not support the intelpython/3.6.8 module.  It is just "there".  They will say use `/contrib/miniconda3/modulefiles`
I agree it is ridiculous to provide a broken, half-baked software.